lib/migration/plugins/init.go

```diff
@@ -20,4 +20,5 @@ import (
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin1"
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin2"
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin5"
+	_ "github.com/vmware/vic/lib/migration/plugins/plugin7"
```

Comment:
we should send a pr to rename those, pluginX is really confusing. It could be the name of the enum for example, _ "github.com/vmware/vic/lib/migration/plugins/VicadminProxyVarRenameVersion"